profiling/src/profiling/mod.rs

```diff
@@ -1114,7 +1122,7 @@ mod tests {
     }
 
     #[test]
-    #[cfg(feature = "timeline")]
+    #[cfg(all(feature = "timeline", not(miri)))]
```

Comment:
This is unrelated, but I was using `cargo +nightly miri` locally to see if the specific nightly version was compatible with the changes, and this fails on miri:

> error: unsupported operation: `extern` static `bindings::ffi::datadog_php_profiling_get_profiling_context` from crate `datadog_php_profiling` is not supported by Miri